storm/src/main/storm/mesos/util/MesosCommon.java

```diff
@@ -60,7 +60,7 @@
   public static final String SUPERVISOR_ID = "supervisorid";
   public static final String ASSIGNMENT_ID = "assignmentid";
   public static final String DEFAULT_WORKER_NAME_PREFIX_DELIMITER = "_";
-  public static final String DEFAULT_MESOS_COMPONENT_NAME_DELIMITER = " | ";
+  public static final String DEFAULT_MESOS_COMPONENT_NAME_DELIMITER = "|";
```

Comment:
@erikdw I took out the spaces because if splitting on '|' it would cause unneeded spaces on both sides